src/hooks/useEvents.ts

```diff
@@ -29,10 +29,9 @@ export const useEvents = () => {
 
             try {
                 setLoading(true);
-
                 while (hasPreviousPage) {
                     const url = startCursor
-                        ? `https://guild.host/api/next/torc-dev/events/past?first=50&after=${startCursor}`
+                        ? `https://guild.host/api/next/torc-dev/events/past?after=${startCursor}?before=${endCursor}`
```

Comment:
```suggestion
                        ? `https://guild.host/api/next/torc-dev/events/past?after=${startCursor}&before=${endCursor}`
```

It would be even better to use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) than manually concatenating a query string for reasons like this.